include/crocoddyl/core/solvers/ddp.hpp

```diff
@@ -29,7 +29,7 @@ class SolverDDP : public SolverAbstract {
   virtual double tryStep(const double& steplength = 1);
   virtual double stoppingCriteria();
   virtual const Eigen::Vector2d& expectedImprovement();
-  virtual double calc();
+  virtual double calcDiff();
```

Comment:
Hi Carlos,

Could we please avoid this type of arbitrary API changes? It breaks some of my code, and is a bit frustrating to find.